plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/impl/PluginModernizer.java

```diff
@@ -15,24 +16,39 @@ public class PluginModernizer {
 
     private final MavenInvoker mavenInvoker;
 
+    private final GHService ghService;
+
     public PluginModernizer(Config config) {
         this.config = config;
         this.mavenInvoker = new MavenInvoker(config);
+        this.ghService = new GHService();
```

Comment:
Config is not needed ? It's useful for the cache path and dry run option

Did we try with `--dry-run` ?